script.js

```diff
@@ -1,62 +1,55 @@
-const imgArr=["./num0.jpg", "./num1.png", "./num2.png", "./num3.png", "./num4.png", "./num5.png", "./num6.jpeg"
-, "./num7.png", "./num8.png", "./num9.jpg", "./num10.jpg", "./num11.png", "./num12.png", "./num13.png"
-, "./num14.png", "./num15.png", "./num16.jpeg", "./num17.png", "./num18.png", "./num19.jpg"];
-var clickedNumArr=[]; 
-var clickedIdArr=[];

```

Comment:
Let's use `let` instead of `var`.

Reply:
Both are meant for the global scope, so what's the point? after some research I understand now that var will force the scope to be global no matter what block I put my variable in, Do you want "let" to avoid bugs when I collaborate with others from now on?

Reply:
Yes, basically, `let` acts more like variable definition in other programming languages, and is more standardly used across the industry. I don't remember the last time I saw anyone use `var` except when they absolutely had to, and it is generally considered bad practice.

See [this answer](https://stackoverflow.com/a/21907082) for details.